packages/cadl-vs/src/VSExtension.cs

```diff
@@ -36,37 +37,51 @@ public sealed class ContentDefinition {
   [Export(typeof(ILanguageClient))]
   [ContentType("cadl")]
   public sealed class LanguageClient : ILanguageClient {
+
     public string Name => "Cadl";
-    public IEnumerable<string>? ConfigurationSections => null;
+    public IEnumerable<string>? ConfigurationSections {
+      get {
+        yield return "cadl";
```

Comment:
nit, but I'd do `{ get; } = new[] { "cadl" }` like FilesToWatch below.

Reply:
yeah, copied that from the sample, did feel weird.